docs/src/tutorial/getting-started.md

```diff
@@ -152,22 +152,32 @@ sudo k8s status
 ```
 You should see `storage enabled` in the command output.
 
-Let's create a persistent volume:
+Let's create a pod with a persistent volume claim:
+
 ```
-sudo k8s kubectl apply -f https://k8s.io/examples/pods/storage/pv-volume.yaml
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/canonical/k8s-snap/main/tests/e2e/templates/storage-setup.yaml
```

Comment:
add a copy of the manifest under the `docs/src` folder (i would recommend `docs/src/manifests`?) and then add a relative link here (e.g. `../../manifests/tutorial-pod-with-pvc.yaml`).

Otherwise the docs will break if we ever update or change our e2e tests.